fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/NOOPFilter.java

```diff
@@ -33,12 +31,7 @@
 public class NOOPFilter implements EventFilter {
 
     @Override
-    public Predicate<Event> getFilter(final Session session) {
-        return this;
-    }
-
-    @Override
-    public boolean apply(final Event event) {
+    public boolean test(final Event event) {
```

Comment:
@ajs6f feel free to remove this class in this PR; otherwise, removing it can be part of a separate PR.